PlasterTemplate/PlasterTemplates/GitLab/psake.ps1

```diff
@@ -268,16 +273,32 @@ Task Deploy -Depends TestAfterBuild {
     "`nInvoking PSDeploy"
     Invoke-PSDeploy @Verbose @Params
 
-    <#
-    "`tSetting git repository url"
-    $GitLabUrl = 'https://{0}@gitlab.com/<%= $PLASTER_PARAM_GitLabUserName %>/{1}.git' -f $ENV:GitLab_PAT, $env:BHProjectName
-
-    "`tPushing built docs to GitLab"
-    git add "$env:BHProjectPath\docs\*"
-    git add "$env:BHModulePath\CHANGELOG.md"
-    git commit -m "Bump version to $ReleaseVersion`n[ci skip]"
-    # --porcelain is to stop git sending output to stderr
-    git push $GitLabUrl HEAD:master --porcelain
-    "`n"
-    #>
+<%
+    if ($PLASTER_PARAM_DeployDocs -eq 'Yes') {
+        '   "`tSetting git repository url"'
+        '   # This may need to be changed if your gitlab server does not use HTTPS'
```

Comment:
These need one more leading space inside the quotes as it's only 3 at the moment